requirements-optional.txt

```diff
@@ -1,3 +1,5 @@
 matplotlib
 pandas
-pyrevolve
\ No newline at end of file
+pyrevolve
+scipy
```

Comment:
Do we need to update any install instructions - what happens when new users try the tutorials? Will they know that these requirements need to be installed?

Reply:
Install instruvtions should still be the same and examples should always be installed with `.[extras]` these were leftover that should have been moved here